pkg/client/didexchange/client.go

```diff
@@ -134,9 +134,9 @@ func (c *Client) QueryConnections(request *QueryConnectionsParams) ([]*Connectio
 	}, nil
 }
 
-// GetConnection fetches single connection record for given id
-func (c *Client) GetConnection(connectionID string) (*ConnectionResult, error) {
-	conn, err := c.connectionStore.GetConnection(connectionID)
+// GetConnectionRecord fetches single connection record for given id
+func (c *Client) GetConnectionRecord(connectionID string) (*ConnectionResult, error) {
```

Comment:
Do we need this change ? GetConnectionRecord returning ConnectionResult looks odd.

Reply:
rolled back ..

Reply:
Agree with @rolsonquadras - looks odd.

Reply:
Is the GetconnectionRecord name is odd or connectionResult ? ..,

Reply:
connectionResult is an odd name (as it simply wraps a ConnectionRecord):

https://github.com/hyperledger/aries-framework-go/blob/3c7a644c101c5c932aae0488be4f7fe7c36df5b0/pkg/client/didexchange/models.go#L46

Reply:
there is // TODO: this model is not final, to be updated as part of #226
above this line, i guess it was randomly picked, Also i am not really sure why we are wrapping. 
We can just return didexchange.ConnectionRecord . Unless @fqutishat had something else invisioned